packages/blocto-sdk/src/providers/ethereum.ts

```diff
@@ -296,6 +296,14 @@ export default class EthereumProvider
   async request(payload: EIP1193RequestPayload): Promise<any> {
     const existedSDK = (window as any).ethereum;
     if (existedSDK && existedSDK.isBlocto) {
+      if (payload.method === 'wallet_switchEthereumChain') {
+        existedSDK.request(payload).then((chainId: string) => {
```

Comment:
It won't return chainId? ref: https://eips.ethereum.org/EIPS/eip-3085

Reply:
My mistake. Fixed in 31449d8